src/SampleWebApp/Startup.cs

```diff
@@ -41,6 +42,11 @@ public Startup(IHostingEnvironment env, IApplicationEnvironment appEnv)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
+            services.AddDbContext<BookDataContext>(options =>
+                    options.UseSqlServer(@"Server=(localdb)\mssqllocaldb;Database=BookDataContext-36f2c634-aa7b-4dbd-80e6-95e06e3521a3;Trusted_Connection=True;MultipleActiveResultSets=true"));
```

Comment:
Wait - did scaffolding generate two different databases for this? 

We would definitely not recommend putting the connection strings for these in code right?